pages/languages/go.md

```diff
@@ -3,3 +3,334 @@ permalink: /languages/go/
 title: Go
 parent: Languages
 ---
+The purpose of this guide is to help familiarize yourself with the Go
+ecosystem and set up a handful of tools to help create a seamless working
+development environment. This guide will help you will learn how to manage Go
+installations from source and manage project dependencies in isolation.
+
+## Go installation
+
+Binary packages for Go are available for Linux, OS X, and Windows platforms,
+but it can be installed from source on even more platforms (even [Plan
+9](https://mike-bland.com/2015/06/08/getting-go-on-plan-9.html)). The ["Getting
+Started" page on golang.org](https://golang.org/doc/install) has the complete
+instructions to install the software and configure the environment. On OS X,
+it can also be installed using [Homebrew](http://brew.sh/).
+
+## Managing versions using `gvm`
+
+On a UNIX-like system (e.g. Linux and OS X), the recommended way to install
+different Go versions is to eschew the earlier methods in favor of a tool
+called [`gvm` (Go Version Manager)](https://github.com/moovweb/gvm). This tool
+automates the installation process (from source by default, or using a binary
+distribution by adding the `-B` switch) and makes it easy to switch between
+multiple Go versions, each in their own [`GOPATH` workspace](#gopath).
+
+## gvm installation and usage
+
+While the `gvm` repository’s `README` has pretty clear and complete
+instructions, the basics of installing and using `gvm` are as follows. First,
+to install the `gvm` program:
+
+```sh
+$ bash < <(curl -s -S -L https://raw.githubusercontent.com/moovweb/gvm/master/binscripts/gvm-installer)
+```
+
+The latest Go version at the time of writing is 1.5.1; you can see the
```

Comment:
I immediately tried to install 1.5.1 based on this and didn't see the caveat below until after I ran into a few errors. Not sure if that's a problem but wanted to note it